graphql/query_executor.go

```diff
@@ -26,7 +27,7 @@ func queryExecute(d *schema.ResourceData, m interface{}, querySource, variableSo
 		return nil, err
 	}
 
-	req, err := http.NewRequest("POST", apiURL, &queryBodyBuffer)
+	req, err := http.NewRequestWithContext(ctx, "POST", apiURL, &queryBodyBuffer)
```

Comment:
The default context used by the SDK when executing the CRUD funcs in a resource is 20 minutes. If this needs to be more configurable, that is also possible by providing more control over the ctx using `{CRUD}WithoutContext` and accessing timeout settings manually.